### PR TITLE
Added korean translation

### PR DESCRIPTION
**Pull request by gisellevonbingen**
:heart:

### CODE REVIEW

File: src/main/resources/assets/systeams/lang/ko_kr.json [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+  "info.systeams.air_max_usage": "최대 사용량",
+  "info.systeams.air_volume_unit": "mL",
+  "info.systeams.air_per_tick": "최대 공기 사용량",
+  "info.systeams.air_per_tick_unit": "mL/틱",
+
+  "block.pneumaticcraft.advanced_pressure_tube.desc": "Shift-사용으로 발전기를 보일러로 변환합니다. 증기 파이프를 회수합니다",
+
+  "block.systeams.steam_dynamo": "증기 발전기",
+  "info.systeams.steam_dynamo.0": "증기를 동력으로 전환합니다.",
+  "info.systeams.steam_dynamo.1": "여기 점점 찜찜해지는데?",
```
Comment (ChiefArug) [inline src/main/resources/assets/systeams/lang/ko_kr.json:28]: If this (or any other) joke/pun doesn't translate well into Korean feel free to replace it with a different (but still related to steam power generation) one that is funny in Korean. Or leave it as it is and make a comment on the PR if you can't come up with another (or if it is already funny enough in Korean).

File: src/main/resources/assets/systeams/lang/ko_kr.json [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+  "block.systeams.numismatic_boiler": "화폐 보일러",
+  "info.systeams.numismatic_boiler.0": "경제적인 가치를 증기로 변환합니다.",
+  "info.systeams.numismatic_boiler.1": "순수한 욕망이 소용돌이 치는 다른 차원의 입구일지도 모릅니다.",
+  "info.systeams.numismatic_boiler.2": "너무 깊게 생각하지 말고 그냥 동전이나 집어넣죠.",
+  "block.systeams.numismatic_boiler.keyword": "동전 엔진 생산기 증기",
+
+  "block.systeams.pneumatic_boiler": "공압 보일러",
+  "info.systeams.pneumatic_boiler.0": "공압을 증기로 변환합니다.",
+  "info.systeams.pneumatic_boiler.1": "어떤사람들은 뜨거운 공기로 동작한다고 말하지만",
+  "info.systeams.pneumatic_boiler.2": "걱정마세요, 여기선 물을 끓이기만 합니다!",
```
Comment (ChiefArug) [inline src/main/resources/assets/systeams/lang/ko_kr.json:73]: Same here

**Review by ChiefArug — COMMENTED** [review]
Looks pretty good from running it back through google translate (just to make sure there isn't anything very blatantly wrong).

File: src/main/resources/assets/systeams/lang/ko_kr.json (hunk 1 above)
Comment (gisellevonbingen) [reply src/main/resources/assets/systeams/lang/ko_kr.json:28]: yeah, that's enough for now.

**Comment by ChiefArug** [comment]
Released on CurseForge now
[1.19.2](https://legacy.curseforge.com/minecraft/mc-mods/systeams/files/4722285)
[1.18.2](https://legacy.curseforge.com/minecraft/mc-mods/systeams/files/4722299)

**Pull request merged**